### DISCUSSION TITLE
Status unhealthy after server reset

### DISCUSSION THREAD

**Discussion by kennylajara** [NginxProxyManager/nginx-proxy-manager#1257] [Q&A] [answered]
Recently my server restarted and now my NPM status health marked as `unhealty`. The proxy is not running, all the pages being proxied went down. I don't even now how to start debugging this.

This is all the information I have and I can think it may be relevant. The option `restart` of the docker compose file was (and still is) `always`. The version was 2.9.3 and now (after it went down) I updated it 2.9.5 to see if it solve the issues. And restarting the container doesn't seems to be helpful.

**Update**: I fount at some point the container exited with code 137 (OMM - Out of memory). I cleared the memory and I even restarted the server but it still happening...

**Comment by kennylajara** [comment, marked as answer]
Solved it by turning down the stack, and turning it up again with a different name (the name of the containers remain the same).

**Comment by johntdavis84** [comment]
Glad it’s working.

Did you try bringing down the stack and bringing it back up without changing the name? That’s usually all I have to do when this happens.

(I’m not so sure this is an NPM thing so much as docker getting confused. It happens to me every once and a while on various non-NPM containers.)

**Reply by kennylajara** [reply]
Yes, I tried it without changing the stack name, I don't know why it didn't work. Also, I'm not sure if it's an NPM or Docker thing. I was using traditional hosting before and I just have about half a year using docker. This is the first time it's happened to me.
It wouldn't be so dramatic for me if it happened to another container, but NPM is the proxy and all the websites went down for about 12 hours. Gosh! I know you don't need to read this, but I had to get it out of my heart.
